src/components/LayerInfoDialog/LayerInfoDialog.vue

```diff
@@ -154,6 +186,23 @@
         await navigator.clipboard.writeText(url)
         alert(`The following url is copied to clipboard! ${ url }`)
       },
+      async getWmsUrl() {
+        const capabilities = await getWmsCapabilities(this.url)
+        const { bbox } = getLayerProperties(capabilities, this.layer)
+        return buildGeoserverUrl({
+          url: this.url,
+          request: 'GetMap',
+          service: 'WMS',
+          version: '1.1.0',
+          format: 'application/openlayers',
+          srs: 'EPSG:4258',
+          layers: this.layer,
+          width: 700,
+          height: 500,
+          bbox: bbox.flat().join(','),
+          styles: '',
+        })
+      },
```

Comment:
In order to resolve the promise this needs to be done here instead of on the loop on the `LayerChooser` component